### DISCUSSION TITLE
Can I detach a tab from a window?

### DISCUSSION THREAD

**Discussion by denppa** [wezterm/wezterm#4080] [Q&A] [answered]
Resizing font in a window with tabs that contains a lot of logs freeze the window, therefore, I want to detach some tabs into their own windows so that font adjustments are window independent and won't affect the window with long logs.

Thank you very much.

**Comment by wez** [comment, marked as answer]
Take a look at: https://wezfurlong.org/wezterm/cli/cli/move-pane-to-new-tab.html
